magento/pwa-studio
# [PWA-479] Extension Files Missing From Packages

## Description

Extension files were missing from published packages and casuing build errors with scaffolded apps. 

## Related Issue





* Closes #2301 
* [[PWA-479](https://jira.corp.magento.com/browse/PWA-479)] Extension Files Missing From Packages

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Run `DEBUG_PROJECT_CREATION=1 packages/create-pwa/bin/create-pwa` to scaffold an app from local packages
2. Verify that you can `yarn install` and `yarn watch`

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
